Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
Why does removing Werror from  libfastjson_la_CPPFLAGS break AC_CHECK_FUNCS?
I can understand that adding -Werror via CFLAGS can have an influence on AC_CHECK_FUNCS, but dropping it should be safe.